src/modules/SimpleSlippageRouter.sol

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import { Math } from "src/utils/Math.sol";
+import { ERC4626 } from "@solmate/mixins/ERC4626.sol";
+import { SafeTransferLib } from "@solmate/utils/SafeTransferLib.sol";
+import { ERC20 } from "@solmate/tokens/ERC20.sol";
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+import { Owned } from "@solmate/auth/Owned.sol";
+import { Cellar } from "src/base/Cellar.sol";
+
+/**
+ * @title Sommelier Simple Slippage Router
+ * @notice A Simple Utility Contract to allow Users to call functions: deposit, withdraw, mint, and redeem with Sommelier Cellar contracts w/ respective specified slippage params.
+ * TODO: Add revokeApproval?
```

Comment:
remove TODO which is done

Reply:
Yep, I was waiting until you had a look at some of the other TODOs I marked. I was then going to remove all TODOs.